src/components/Card/README.md

```diff
@@ -0,0 +1,55 @@
+# Card
+
+Card is content container. It supports multiple types: `selection`, `action`, `container`.
+
+## Properties
+
+| Property  | Type                                                | Required | Default       | Description                                                         |
+| :-------- | :-------------------------------------------------- | :------: | :------------ | :------------------------------------------------------------------ |
+| children  | `ReactNode`                                         |    ✓     |               | Card's content                                                      |
+| type      | `CardType`                                          |          | `'container'` | Card's type affects on available properties                         |
+| view      | `SelectionCardView` \| `ContainerCardView`          |          | `'outlined'`  | Available for `type`: `'container'` and `'selection'`               |
+| theme     | `ContainerCardTheme`                                |          | `'normal'`    | Card's base color. Available for `type`: `'container'`              |
+| className | `String`                                            |          |               | CSS class                                                           |
+| onClick   | `(event: React.MouseEvent<HTMLDivElement>) => void` |          |               | Card click handler. Available for `type`: `'selection'`, `'action'` |
+| selected  | `Boolean`                                           |          |               | Selected card. Available for type: `'selection'`                    |
+| disabled  | `Boolean`                                           |          |               | Disabled card. Available for type: `'selection'`, `'action'`        |
+
+### Typings
+
+```typescript
+type CardType = 'selection' | 'action' | 'container';
+
+type SelectionCardView = 'outlined' | 'clear';
+type ContainerCardView = 'outlined' | 'filled' | 'raised';
+
+type ContainerCardTheme = 'normal' | 'info' | 'positive' | 'warning' | 'danger';
```

Comment:
Same here, component exports `CardTheme`

Reply:
Yes, however, it is easier to know the types of props, since SelectionCardView or ContainerCardView exists in the table above. I can delete this if you want